app/assets/stylesheets/theme/_navbar.scss

```diff
@@ -1,3 +1,8 @@
 .navbar-brand {
   padding: 10px 15px;
 }
+
+.navbar {
+  background-color: $white;
```

Comment:
Usually when you're styling Bootstrap components there's a variable to override https://github.com/adamish717/steam-hours/blob/global-add/app/assets/stylesheets/theme/_bootstrap-overrides.scss#L363